cayenne-server/src/test/java/org/apache/cayenne/access/DataContextFlattenedAttributesIT.java

```diff
@@ -233,87 +234,143 @@ public void testSelectColumnQuery() throws Exception {
         ColumnSelect<CompoundPaintingLongNames> originalQuery = ObjectSelect.query(CompoundPaintingLongNames.class)
                 .column(CompoundPaintingLongNames.SELF);
 
-        CompoundPaintingLongNames beforeCompoundPainting = originalQuery.selectFirst(context);
+        CompoundPaintingLongNames beforeCompoundPainting = originalQuery.select(context)
```

Comment:
I think it could be easier to update test code, to be sure we have single entry in the DB. Something like `tArtist.deleteAll(); tPainting.deleteAll();` calls before test data creation should do the trick.